client/landing/domains/registrant-verification/index.jsx

```diff
@@ -203,6 +218,7 @@ class RegistrantVerificationPage extends Component {
 				break;
 
 			case 'KS_RAM_error':
+			case 'KS_RSP_error':
```

Comment:
This is a semi-related change - without it, RSP domains (our reseller account) would not be recongized properly here :(